packages/flutter/lib/src/services/text_input.dart

```diff
@@ -483,6 +484,9 @@ class TextInputConfiguration {
   }) : smartDashesType = smartDashesType ?? (obscureText ? SmartDashesType.disabled : SmartDashesType.enabled),
        smartQuotesType = smartQuotesType ?? (obscureText ? SmartQuotesType.disabled : SmartQuotesType.enabled);
 
+  /// The ID of the view that the text input belongs to.
```

Comment:
It's not immediately clear to me that this is referring to the multi-view/window project. Maybe that will be more obvious once the feature is finished and people are more familiar with it, but if possible, can you link `[View]` or something like that?

Reply:
Done